packages/web/components/templates/article/ArticleContainer.tsx

```diff
@@ -228,11 +243,6 @@ export function ArticleContainer(props: ArticleContainerProps): JSX.Element {
           '@md': {
             maxWidth: 1024 - styles.margin,
```

Comment:
Maybe better to just use `appEmbedMaxWidthPercentage` here too. Could work better for iPad.